core-api/src/services/checkoutSlot.service.ts

```diff
@@ -0,0 +1,57 @@
+import db from "../config/database";
+import { Place } from "../entities/place.entity";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import { SafeCheckoutSlot, CheckoutSlot } from "../entities/checkoutSlot.entity";
+import messages from "../utils/messages";
+
+/**
+ * @namespace Services
+ * @class PlaceService
+ */
+export class CheckoutSlotService {
+
+    public static async findAll(): Promise<SafeCheckoutSlot[]> {
+        const repository = db.getRepository(CheckoutSlot);
+
+        try {
+            const slots: CheckoutSlot[] = await repository.find({ relations: ["place"] });
+            return slots.map(slot => slot.toSafe());
+        } catch (e) {
+            throw new ServerError(e.message, ErrorCode.DATABASE_ERROR);
+        }
+    }
+
+    public static async findByPlace(placeId: number): Promise<SafeCheckoutSlot[] | null> {
```

Comment:
Don't you think that this function will have more sense if moved to placeServer? Because this is actually a feature from places, just like find meals by placeId.

Reply:
There is a method in PlaceService that calls this one.

Reply:
Ok, this way make sense. I didn't see this call method in PlaceService.